ui/fast_forward_button.js

```diff
@@ -70,10 +73,10 @@ shaka.ui.FastForwardButton = class extends shaka.ui.Element {
     }
 
     const trickPlayRate = this.player.getPlaybackRate();
-    // Every time the button is clicked, the rate is multiplied by 2,
-    // unless the rate is at max (8), in which case it is dropped back to 1.
-    const newRate = (trickPlayRate < 0 || trickPlayRate > 4) ?
-        1 : trickPlayRate * 2;
+    const newRateIndex = this.fastForwardRates_.indexOf(trickPlayRate) + 1;
+
+    const newRate = (newRateIndex != this.fastForwardRates_.length) ?
```

Comment:
Would you like to add a comment here, similar to the comment we had before, to explain how the play rate changes work?